python/lance/pytorch/data.py

```diff
@@ -30,23 +31,29 @@
 
 import lance
 from lance import dataset
+from lance.types.image import is_image
```

Comment:
I was thinking that if something is public api for types it should be imported in types/__init__.py and imported as "from lance.types import is_image". Wdyt?

Reply:
I shared the same sentiment with you. So for now, i am not sure this is_image will be a public API or private API, that's why i did not expose it via `lance.types`. But if we think it is possible for users to use it. i can add it to `lance.types.__init__` module.